src/components/Header.tsx

```diff
@@ -1,33 +1,42 @@
 import React, { Dispatch, SetStateAction } from 'react';
-// @ts-ignore
+
 import logo from '../image/logo.png';
 
 interface IHeader {
-  setSignInModalOn: Dispatch<SetStateAction<boolean>>;
-  setSignUpModalOn: Dispatch<SetStateAction<boolean>>;
+  setSetTitleModal: Dispatch<SetStateAction<string>>;
+  setIsOpen: Dispatch<SetStateAction<boolean>>;
 }
 
-export function Header({ setSignInModalOn, setSignUpModalOn }: IHeader) {
-  const onClickSignInHandler = () => {
-    setSignInModalOn(true);
-  };
-  const onClickSignUpHandler = () => {
-    setSignUpModalOn(true);
+export function Header({ setSetTitleModal, setIsOpen }: IHeader) {
```

Comment:
I found the previous version of this component's API to be more elegant. 
`<Header onSignInButtonClick={...} onSignUpButtonClick={...}/>`